### ISSUE TITLE
No results in control panel?

### ISSUE THREAD

**Issue by Coalescent** [putyourlightson/craft-entry-count#26]
I assume I'm supposed to see a summary of entries and counts in the Entry Count cp area - but I just see "No entries counted".

I am incrementing counts - and have confirmed that from the front end via Twig (ie. `{{ craft.entryCount.count(entry.id) }}` but none of that is reflected in the CP.

Craft 3.4.4
MySQL 5.7.26
PHP 7.3.8
Entry Count 2.0.2

**Comment by bencroker** [comment]
So does this code display a running total of the count?

```twig
{% do craft.entryCount.increment(entry.id) %}

{{ craft.entryCount.count(entry.id) }}
```

If it does, but nothing appears in the control panel, then maybe reinstall the plugin as something must not have worked in the initial install.

**Comment by Coalescent** [comment]
Yep, it does... I have it showing the running count for a set of entries as I view them.

Uninstall/reinstall and still the same behaviour.

**Comment by bencroker** [comment]
The plugin uses the `ElementQuery::EVENT_BEFORE_PREPARE` event. Do you have any custom modules/plugins installed that might be interfering with this?

You can also try this code in a template to see if any counted entries are returned.

```twig
{{ craft.entryCount.countedEntries.count() }} entries counted
```

Can you also check that the `entrycount` datbase table is being populated?

**Comment by Coalescent** [comment]
> The plugin uses the ElementQuery::EVENT_BEFORE_PREPARE event. Do you have any custom modules/plugins installed that might be interfering with this?

I'm not certain... we don't have any custom plugins we've written doing that but I'm not certain about other third party plugins. I did a quick search for the string `EVENT_BEFORE_PREPARE` and got nothing so probably not.

> {{ craft.entryCount.countedEntries.count() }} entries counted

When I do this, I get this error:

`Neither the property "countedEntries" nor one of the methods "countedEntries()", "getcountedEntries()"/"iscountedEntries()"/"hascountedEntries()" or "__call()" exist and have public access in class "putyourlightson\entrycount\variables\EntryCountVariable".`

> Can you also check that the entrycount datbase table is being populated?

Yes, it appears to be populating correctly with the correct entry id and corresponding counts.

**Comment by bencroker** [comment]
Hmm, strange, what about this?

```twig
{{ craft.entryCount.getCountedEntries().all()|length }} entries counted
```

**Comment by Coalescent** [comment]
Similar error:

`Neither the property "getCountedEntries" nor one of the methods "getCountedEntries()", "getgetCountedEntries()"/"isgetCountedEntries()"/"hasgetCountedEntries()" or "__call()" exist and have public access in class "putyourlightson\entrycount\variables\EntryCountVariable".`

**Comment by bencroker** [comment]
Sounds like there must be a problem with your installed version. Can you remove the plugin files completely and reinstall?

**Comment by Coalescent** [comment]
I did uninstall it and reinstall it before via the cp. Is there something more I can do after uninstalling? ie. Are the plugin files still there to delete manually (I've never checked before)?

**Comment by bencroker** [comment]
There’s an option to “remove” the plugin after uninstalling in the CP which will remove the files too. Otherwise you can use `composer remove` as well.

**Comment by Coalescent** [comment]
Thanks Ben but still the same behaviour after removing and installing again.

I'm not a plugin developer so this comes a bit more from a layman's perspective but when I look at your plugin files, I can't even find a method for `getCountedEntries` or `countedEntries`... so maybe I'm missing something but I don't understand why you asked me to try those? Are you saying those methods exist but for some reason my installed files are missing them?

**Comment by bencroker** [comment]
Apologies Mike, I was referencing a different version of the plugin. The method is called `getEntries` in version 2.0.2, so this is the code:
```
{{ craft.entryCount.getEntries().count() }} entries counted
```

**Comment by Coalescent** [comment]
Ok, so that works and returns the expected result... as does this in a front end template:

`{% set entries = craft.entryCount.entries.all() %}`

But that returns nothing (ie. an empty array) when it is called in your cp template at `craft/vendor/putyourlightson/craft-entry-count/src/templates/index.html`

**Comment by Stefenyo** [comment]
I'm getting this as well. Not showing up in the backend. I'm running the latest version of craft.

**Comment by bencroker** [comment]
Do you have a multi-site setup? Could it be that the front-end site that it is working on is not the primary site?

**Comment by Stefenyo** [comment]
No multi-site setup here. I did a fresh install of the plugin then I even did a fresh install of craft cms, unfortunately still getting the same thing.

**Comment by Coalescent** [comment]
Mine is a multi - and yes, it is on a secondary site... but in light of @SweetPoStayPro's non-multi, I guess that's not the issue? I could try testing on the primary... ok, just tested it. I get results on primary site entries in the CP - but not the secondary.

**Comment by bencroker** [comment]
@Coalescent Ok so the issue is multi-site, I'll see if I can release a fix for this soon, thanks for sticking with it!

@SweetPoStayPro Can you please provide some more info and show your code?

**Comment by Coalescent** [comment]
Same to you Ben and thanks for providing a fix!

**Comment by bencroker** [comment]
Released a fix in v2.0.3, however note that the plugin has limited multi-site compatibility, as it is intended for educational purposes. For anything more complex, check out the [View Count](https://www.doublesecretagency.com/plugins/view-count) plugin.
